cEP-0005.md

```diff
@@ -102,18 +102,25 @@ class RedundancyBear(LocalBear):
     # This bear is supposed to be able to fix unused imports...
     FIX_ASPECTS = {Root.Redundancy.UnusedImport}
     # ... and detect code clones.
-    DETECT_ASPECTS = {Root.Redundancy.CloneRedundancy}
+    DETECT_ASPECTS = {Root.Redundancy.Clone}
 
     # Aspects are passed as parameter
     def run(self, filename, file, aspects):
-        # No documentation required anymore for the bears and their parameters.
+        # No documentation required anymore for the bears.
 
-        # Check the code ...
+        if unused_imports(file):
+            # A bear may yield results for any aspect even if it's not selected.
+            # coala will filter out only selected aspects.
```

Comment:
nope, the bear may or may not use the aspects to perf optimize, it may just yield everything. Convenience feature esp. useful for linter bears